rviz_default_plugins/include/rviz_default_plugins/displays/robot_model/robot_model_display.hpp

```diff
@@ -115,7 +115,9 @@ private Q_SLOTS:
    * "Robot Description" property, iterates through the links, and
    * loads any necessary models. */
   virtual void load_urdf();
-  virtual void load_urdf_from(const std::string & filepath);
+  virtual void load_urdf_from_file(const std::string & filepath);
```

Comment:
This breaks API, so I don't know if it is a good idea to have it backported.

Reply:
@Karsten1987 looking at this from the outside, it's unclear if there is an API-compatible way to resolve the desired issue. It looks like it will require more effort than a strict cherry-pick. Are you willing to commit to that work or should we drop this from the release?

Reply:
I don't really think there's a straightforward way wo/ breaking API.
I am ok with closing this PR and release the fix with dashing.

Reply:
> I don't really think there's a straightforward way wo/ breaking API.

Couldn't you just leave the function name as-is (maybe have it call the new `load_urdf_from_file`? Or is it that the behavior of the function has changed?

Reply:
we could do that if we are fine with breaking ABI compatibility. Behavior hasn't changed (except from the new functionality of loading the URDF from XML content string).

thoughts?

Reply:
latest commit https://github.com/ros2/rviz/pull/382/commits/966a33cf0807fd0447b1624c9c84c8d410b9a36b keeps crystal API compatible.

Reply:
Why does that break ABI? Because of new virtual methods?